avoid hardcoding specific versions for dependencies of moviepy, there should be no need to
author: Kenneth Hoste (HPC-UGent)
--- moviepy-0.2.3.2/setup.py.orig	2017-12-18 22:18:16.169427000 +0100
+++ moviepy-0.2.3.2/setup.py	2017-12-18 22:19:01.375171000 +0100
@@ -60,7 +60,7 @@
 exec(open('moviepy/version.py').read()) # loads __version__
 
 # Define the requirements for specific execution needs.
-requires = ['decorator==4.0.11', 'imageio==2.1.2', 'tqdm==4.11.2', 'numpy']
+requires = ['decorator', 'imageio', 'tqdm', 'numpy']
 optional_reqs = ['scikit-image==0.13.0', 'scipy==0.19.0', 'matplotlib==2.0.0']
 documentation_reqs = ['pygame==1.9.3', 'numpydoc>=0.6.0',
     'sphinx_rtd_theme>=0.1.10b0', 'Sphinx>=1.5.2'] + optional_reqs
